astropy/io/fits/connect.py

```diff
@@ -165,6 +165,10 @@ def read_table_fits(input, hdu=None):
     for key, value, comment in table.header.cards:
 
         if key in ['COMMENT', 'HISTORY']:
+            # Convert to astropy's format
```

Comment:
@saimn Oh I meant `astropy's convention`. It that better or `io.ascii format` better?